kernel/libs/aster-bigtcp/src/lib.rs

```diff
@@ -12,6 +12,10 @@
 
 #![no_std]
 #![deny(unsafe_code)]
+// Rustdoc does not render methods of public type aliases that reference private types,
```

Comment:
```suggestion
// Rustdoc does not render methods of public type aliases that reference sealed types,
```

Sorry, my previous wording was not precise. "Sealed" is a more accurate term than "private."